scrapers/heart.py

```diff
@@ -41,7 +44,13 @@ def scrape_heart():
         description = coffee_soup.find('div', {'class':'tab-content small'}).find('div',{'id': 'tab1'}).text.encode('utf-8').strip()
         notes = coffee_soup.find('p',{'class': 'small uppercase flavors'}).text.split(',')
         if not blend:
-            region = coffee_soup.find('div', {'class':'tab-content small'}).find('div',{'id': 'tab1'}).p.text.replace(u'Location:\xa0', '').replace('Location:', '').encode('utf-8')
+            # region = coffee_soup.find('div', {'class':'tab-content small'}).find('div',{'id': 'tab1'}).p.text.replace(u'Location:\xa0', '').replace('Location:', '').encode('utf-8')
+            is_country_in_here = [x for x in countrydict.keys() if x in name.lower()]
```

Comment:
can you make this thing into a function and put it in helpers.py since we use it elsewhere and is helpful?